script.js

```diff
@@ -60,10 +60,17 @@ function storeNonEmptyCurrentDisplay() {
 
 
 function evaluateNow() {
-    if (numberArray.length === 1) ans = numberArray[0];
+    if (numberArray.length === 1 && !operatorSet) ans = numberArray[0];
```

Comment:
operatorSet defaulted to false. Add this extra condition to prevent -= interfered by one number -> =